### PR TITLE
Mg stable diffusion xl

### PR DESCRIPTION
**Pull request by bingatgoogle**
<br>
Add the notebook for Stable Diffusion XL models.
<br><br><br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [X] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb [hunk 1]
```diff
@@ -0,0 +1,610 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>missing link to run in workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>I would drop the specific reference to Colab</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Fill -&gt; Set the</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Vertex-AI API -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>describe what the functions do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>the Endpoint -&gt; to a Vertex AI Endpoint resource</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (bingatgoogle) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Done.</p>

Comment (bingatgoogle) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Done.</p>

Comment (bingatgoogle) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Done.</p>

Comment (bingatgoogle) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Done.</p>

Comment (bingatgoogle) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Done.</p>

Comment (bingatgoogle) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb:1]: <p>Added the workbench link.</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**